flyway/postgre/migrations/sql/V1_3_Create_lesson2_tables.sql

```diff
@@ -0,0 +1,85 @@
+-- Database: admindb
+
+-- DROP DATABASE IF EXISTS admindb;
+
+CREATE TYPE sex AS ENUM ('female', 'male');
+CREATE TYPE status AS ENUM ('active', 'deactived', 'deleted');
+
+DROP TABLE IF EXISTS sso_organization;
+CREATE TABLE sso_organization (
```

Comment:
Không nên xóa bảng vì có thể gây mất dữ liệu. Chỉ cần câu lệnh CREATE TABLE IF NOT EXISTS thì sẽ không bị lỗi khi bảng đã tồn tại.
```suggestion
CREATE TABLE IF NOT EXISTS sso_organization (
```